### DISCUSSION TITLE
Dashboard header misaligned

### DISCUSSION THREAD

**Discussion by m4kman** [folke/snacks.nvim#341] [Q&A] [answered]
Hey, 

So I'm using snacks.nvim to define a custom header for my dashboard, but it seems to be misaligned even though it looks fine in my config. I've attached screenshots for reference. Does anyone know how I can fix this? 

Config (how the header should look like):
![Screenshot 2024-12-18 at 10 29 50 AM](https://github.com/user-attachments/assets/abf75cfd-939a-40e3-92c0-c0b2f0048859)

Dashboard: 
![Screenshot 2024-12-18 at 10 33 28 AM](https://github.com/user-attachments/assets/78e024b1-c5b0-463b-ba8e-36a429bc264a)

Finally, here's my snacks.lua: 
![Screenshot 2024-12-18 at 10 34 27 AM](https://github.com/user-attachments/assets/5a2d1fdc-b9bc-4ca1-80b4-277deaec747a)

_Quick notes: This isn't specific to the ```advanced``` example, rather it persists across the default example as well._

**Comment by Jas-SinghFSU** [comment]
Make sure there is no space from the starting column. In other words, make sure the bottom-left part of the N in NeoVim touches the left edge of the screen. And maybe move the square brackets up a line at the end of the m in NeoVim. I've noticed that having spacing in my header makes it buggy.

**Reply by m4kman** [reply]
Thanks for responding. I've tried what you suggested and it still doesn't work. 

![Screenshot 2024-12-18 at 11 20 43 AM](https://github.com/user-attachments/assets/b878678e-bb98-4d21-aeb5-117c1e454cdc)

**Reply by jcomesana** [reply, marked as answer]
I was playing with a header and I had this very same problem. To get an aligned header, all the lines need to have the same length, which means that you might need to add spaces at the end.

**Reply by m4kman** [reply]
You, sir, are a legend. Much appreciated.
